requirements/quality.txt

```diff
@@ -102,6 +102,7 @@ sphinx==1.6.3             # via edx-sphinx-theme
 sphinxcontrib-napoleon==0.6.1
 sphinxcontrib-websupport==1.0.1  # via sphinx
 stevedore==1.25.0         # via doc8
+testfixtures==4.5.0
```

Comment:
Add requirements to appropriate ".in" file and run `make upgrade` to update `.txt` files